src/main/java/anticope/rejects/gui/screens/StatsScreen.java

```diff
@@ -49,9 +49,9 @@ private void updateData() {
             liv.getActiveStatusEffects().forEach((effect, instance) -> {
                 String status = lang.get(effect.getTranslationKey());
                 if (instance.getAmplifier() != 0) {
-                    status += (String.format(" %d (%s)", instance.getAmplifier()+1, StatusEffectUtil.getDurationText(instance, 1)));
+                    status += (String.format(" %d (%s)", instance.getAmplifier()+1, StatusEffectUtil.getDurationText(instance, 1, 20)));
                 } else {
-                    status += (String.format(" (%s)", StatusEffectUtil.getDurationText(instance, 1)));
+                    status += (String.format(" (%s)", StatusEffectUtil.getDurationText(instance, 1, 20)));
```

Comment:
Probably safe to assume 20 unless we want this to scale on tick warped servers for whatever reason

Reply:
I think its safer to use `TickRate.INSTANCE.getTickRate()`, some servers may run with <20 tps.